src/UIState/SeeDeviceAddress.cpp

```diff
@@ -28,15 +28,33 @@ void SeeDeviceAddress::handleKey(char key) {
   };
 }
 
+int SeeDeviceAddress::writeToBuffer(char* buffer, int size, int line) {
```

Comment:
* Refactor to use a ENUM to identify the type of display instead of the size of the buffer.
* Refactor to identify the information displayed to use more appropriate term instead of "line".